t.sentinel.import/t.sentinel.import.py

```diff
@@ -43,19 +43,88 @@
 #% guisection: Settings
 #%end
 
+#%flag
+#% key: e
+#% description: Use ESA-style scenename/s to download from USGS
+#%end
+
+#%flag
+#% key: a
+#% description: run atmospheric correction with sen2cor before importing
+#%end
+
 #%option G_OPT_F_INPUT
 #% key: settings
+#% required: yes
 #% label: Full path to settings file (user, password)
 #%end
 
+#%option
+#% key: clouds
+#% type: integer
+#% description: Maximum cloud cover percentage for Sentinel scene
+#% required: no
+#% guisection: Filter
+#% answer: 100
+#%end
+
+#%option
+#% key: producttype
+#% type: string
+#% description: Sentinel product type to filter
```

Comment:
```suggestion
#% description: Sentinel-2 product type to filter
```

... unless also S1 will be supported (?)